web/api/netdata-swagger.yaml

```diff
@@ -225,182 +319,266 @@ paths:
+            The grouping method. If multiple collected values are to be grouped
+            in order to return fewer points, this parameters defines the method
+            of grouping. methods are supported "min", "max", "average", "sum",
+            "incremental-sum". "max" is actually calculated on the absolute
+            value collected (so it works for both positive and negative
+            dimesions to return the most extreme value in either direction).
           required: true
           type: string
-          enum: [ 'min', 'max', 'average', 'median', 'stddev', 'sum', 'incremental-sum' ]
-          default: 'average'
+          enum:
+            - min
+            - max
+            - average
+            - median
+            - stddev
+            - sum
+            - incremental-sum
+          default: average
           allowEmptyValue: false
         - name: options
           in: query
-          description: 'Options that affect data generation.'
+          description: Options that affect data generation.
           required: false
           type: array
           items:
             type: string
-            enum: [ 'abs', 'absolute', 'display-absolute', 'absolute-sum', 'null2zero', 'percentage', 'unaligned' ]
+            enum:
+              - abs
+              - absolute
+              - display-absolute
+              - absolute-sum
+              - null2zero
+              - percentage
+              - unaligned
             collectionFormat: pipes
-          default: ['absolute']
+          default:
+            - absolute
           allowEmptyValue: true
         - name: label
           in: query
-          description: 'a text to be used as the label'
+          description: a text to be used as the label
           required: false
           type: string
-          format: 'any text'
+          format: any text
           allowEmptyValue: true
         - name: units
           in: query
-          description: 'a text to be used as the units'
+          description: a text to be used as the units
           required: false
           type: string
-          format: 'any text'
+          format: any text
           allowEmptyValue: true
         - name: label_color
           in: query
-          description: 'a color to be used for the background of the label'
+          description: a color to be used for the background of the label
           required: false
           type: string
-          format: 'any text'
+          format: any text
           allowEmptyValue: true
         - name: value_color
           in: query
-          description: 'a color to be used for the background of the label. You can set multiple using a pipe with a condition each, like this: color<value|color>value|color:null The following operators are supported: >, <, >=, <=, =, :null (to check if no value exists).'
+          description: >-
+            a color to be used for the background of the label. You can set
```

Comment:
Please, uppercase for `a` in the beginning.

Reply:
Fixed.